backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/components/CassandraModule.java

```diff
@@ -185,6 +185,8 @@ static CassandraModule aggregateModules(Collection<CassandraModule> modules) {
             .build();
     }
 
+    CassandraModule NO_MODULE = builder().build();
```

Comment:
Refactoring proposal: make `CassandraClusterExtension` accept a vararg of modules and under the hood call the aggregate method.

Then:
 - We can simplify tests calling CassandraModule.aggregateXXX
 - We can specify an empty varargs without using `CassandraModule.NO_MODULE` which is badly named.

If you onna do this in another PR, then please at least rename this in `EMPTY_MODULE`